[7.x][DOCS] Add a known issue (#96123) to 7.12 release notes (#99617)
elastic/kibana: docs/CHANGELOG.asciidoc

[[known-issues-v7.12.0]]
=== Known issues

> *Saved object migrations*

Author: Formatting probably needs help. I tried to make these subheadings `====` but build didn't allow

If your upgrade migration fails repeatedly, refer to {kibana-ref}/upgrade-migrations.html#preventing-migration-failures[preventing migration failures].

> *Cluster performance degradation due to {kib} search sessions in non-default space (fixed in 7.12.1)*

Reviewer: ```suggestion
*Cluster performance degradation*
```

> Kibana fails to update and delete {kibana-ref}/search-sessions.html[search sessions objects] created in non-default space, degrading the performance of the cluster due to a flooding of async search API usage.

Reviewer: ```suggestion
Details:

{kib} fails to update and delete {kibana-ref}/search-sessions.html[search sessions objects] created in non-default spaces.

Impact:
Cluster performance degrades due to a flooding of async search API usage.
```

> This issue is fixed in 7.12.1 release by {kibana-pull}96123[#96123]. For 7.12.0 workaround refer to {kibana-issue}96124[#96124].

Reviewer: ```suggestion
For information on the workaround for 7.12.0, refer to {kibana-issue}96124[#96124].

For information on the fix in 7.12.1, refer to {kibana-pull}96123[#96123]. 
```
